resources/asciidoctor/spec/inline_callout_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'inline_callout/extension'
+
+RSpec.describe InlineCallout do
+  it "enables support for inline callouts if requested" do
+    attributes = {
+      'inline-callouts' => '',
```

Comment:
What meaning does `''` have as a value? In many languages that value is false-y.

Reply:
In asciidoctor flags attributes are usually controlled by being defined at all. It *is* weird, but it feels more like the way the rest of asciidoctor works.